week3.js

```diff
@@ -0,0 +1,131 @@
+//String
+let myString = "Hello,this,is,a,difficult-,to-,read,sentence";
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/,/g, " ");
+console.log(myString);
+
+//Arrays
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+console.log(favoriteAnimals);
+
+let favoriteAnimalsOfMauro = ["blowfish", "capricorn", "giraffe", "turtle"];
+console.log(favoriteAnimalsOfMauro);
+
+let favoriteAnimalsOfJim = ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"];
+console.log("The favorite animal of Jim is added to the array.");
+console.log(favoriteAnimalsOfJim);
+console.log("The array has a length of:", favoriteAnimalsOfJim.length);
+
+let favoriteAnimalsOfJason = ["blowfish", "meerkat", "capricorn", "turtle"];
+console.log(favoriteAnimalsOfJason);
+console.log(favoriteAnimalsOfJason.length);
+console.log(favoriteAnimalsOfJason[1], "The item you are looking for is at index:2");
+

```

Comment:
This 'Arrays' section needs to be changed. I guess you did not understand the instructions correctly. For this section, the idea was to find array methods that allow you to add/remove/insert items to the array, meaning you don't have to write those arrays yourself but let the array methods do it.  Start, for example by learning push() method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
Thanks for reviewing, gonna check it again

Reply:
Changed to: let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
console.log(favoriteAnimals.push("turtle"));
console.log(favoriteAnimals);

let addedAnimalOfJim = ["blowfish", "capricorn", "giraffe", "turtle"];
console.log(addedAnimalOfJim.splice(1, 0, "meerkat"));
console.log("The new value will be: ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"]");
console.log(addedAnimalOfJim);
console.log("The array has a length of:", addedAnimalOfJim.length);

let deletedAnimalOfJason = ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"];
console.log(deletedAnimalOfJason.splice(-2, 1,));
console.log(deletedAnimalOfJason.length);
console.log(deletedAnimalOfJason[1], "The item you are looking for is at index:2");

Reply:
Check out this array method, see how you can use it in your code https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf